framework/logging.cpp

```diff
@@ -1778,35 +1782,35 @@ void TapFormatLogger::print(int tc)
     case TestFailed:
```

Comment:
Not sure we should `[[fallthrough]]` here from `TestPassed`. If `TestFailed` just breaks,, compiler should be able to "fallthrough it" for us.